sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.csproj

```diff
@@ -11,11 +11,11 @@
 
   <ItemGroup>
     <PackageReference Include="Apache.Avro" />
-    <!--<PackageReference Include="Azure.Data.SchemaRegistry" /> -->
+    <!-- <PackageReference Include="Azure.Data.SchemaRegistry" /> -->
```

Comment:
A lot of whitespace moving around here 😆

Reply:
Yeah I tried to switch the package ref back but it didn't work so I made a different PR that I will circle back to after this

https://github.com/Azure/azure-sdk-for-net/pull/33872

There is some weird stuff happening over there

Reply:
Looks like the Avro encoder tests are failing - https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2173742&view=ms.vss-test-web.build-test-results-tab&runId=39270351&resultId=100122&paneView=debug